eth-connector/src/deposit_event.rs

```diff
@@ -237,7 +237,6 @@ pub struct DepositedEvent {
     pub sender: Address,
     pub token_message_data: TokenMessageData,
     pub amount: Balance,
-    pub fee: Fee,
```

Comment:
But you left 
```rust
EventParam {
    name: "fee".to_string(),
    kind: ParamType::Uint(256),
    indexed: false,
},
```
Is it intentional?

Reply:
It's EthEvent field used for `TokenMessageData` for `ft_on_transfer`